recipe/meta.yaml

```diff
@@ -20,10 +20,15 @@ requirements:
     - pip
   run:
     - python >=3.6
```

Comment:
It may be worth double checking but I am not sure 3.6 is still supported. I think the oldest tesyed was 3.8.

Reply:
I'm making the change to pin/constrain >= 3.8